src/components/plugin/plugin-app.tsx

```diff
@@ -135,9 +139,10 @@ export default class PluginApp extends React.Component<IProps, IState> {
           // ensure the second language set by the teacher is available in the translations before setting
           // and add per-student recording toggle set by the teacher
           this.setState({
-            lang: DEFAULT_LANG,
+            lang: showSecondLanguageFirst && translations[preferredLanguage] ? preferredLanguage : DEFAULT_LANG,
             secondLanguage: translations[preferredLanguage] ? preferredLanguage : undefined
```

Comment:
I would think this would need to return `undefined` if `showSecondLanguageFirst` is true, otherwise won't it show the second language twice?

Reply:
no, i thought so too at first, and it does look funny, but `lang` is just a variable that we use to determine which translation we are currently looking at -- the languages that appear on the toggle are set by this.languages, on lines 93-110.